plugin/metrics/prometheus/metricsstore/reader_test.go

```diff
@@ -357,6 +358,49 @@ func TestGetRoundTripper(t *testing.T) {
 	}
 }
 
+func TestGetRoundTripperToken(t *testing.T) {
+	tokenFilePath := "test/test.txt"
+	wantBearer := "token from file"
+	logger := zap.NewNop()
```

Comment:
nit: I just realised that `logger` is only used if TLS is enabled, which is not the case for this test. So we can remove this and just pass `nil` into `getHTTPRoundTripper`, to simplify the test a little.

Reply:
Done